doc/en/components/charts/chart-features-animations.md

```diff
@@ -7,4 +7,61 @@ mentionedTypes: []
 
 # $PlatformShort$ Chart Animations
 
-<!-- category-chart-config-options.md -->
\ No newline at end of file
+Animations allows you to ease-in the series as it loads a new data source. The available animation differs depending on the type of series involved. For example, the column series animates by rising from the x-axis, a line series animates by drawing from the y-axis.
+
+<code-view style="height: 500px" 
+           data-demos-base-url="{environment:dvDemosBaseUrl}" 
+           iframe-src="{environment:dvDemosBaseUrl}/charts/category-chart-line-chart-with-animations" 
+           alt="$PlatformShort$ Configuration Options Example" 
+           github-src="charts/category-chart/line-chart-with-animations">
+</code-view>
+
+<div class="divider--half"></div>
+
+## Transition In Animations
+
+By default, transition in animations are disabled and you can enable transition-in animations by setting the `IsTransitionInEnabled` option to “true”.
+
+Transition in animations can be configured in terms of transition type, speed of the individual data points relative to each other, and the easing transition (managed by an easing function). The chart also resizes the title and subtitle information.
+
+The transition type is configured by setting the transitionInMode option to the desired transition name. Automatic selection of the transition type based on the series type is possible, too, by setting the property to `Auto` (default).
+
+The following code example demonstrates how to enable transition in animations and have the chart accordion in from the right.
+
+```razor
+<CategoryChart Height="500px" Width="700px"
+    DataSource="@DataSource"
+    IsTransitionInEnabled="true"
+    TransitionInDuration="5000"
+    TransitionInMode="CategoryTransitionInMode.AccordionFromRight"/>
+```
```

Comment:
in all code snippets, change TransitionInDuration to 1000 because 5000 is too high value for animations. We do not want to show very slow animation.

Reply:
TransitionInDuration is not set to 1000 in all code snippets

Reply:
pending